hr_holidays_public/models/resource_calendar.py

```diff
@@ -44,6 +45,9 @@ def _attendance_intervals_batch(
             tz=tz,
             lunch=lunch,
         )
+        if resources is None:
+            resources = [self.env["resource.resource"]]
+
```

Comment:
Still an empty line here.

Reply:
fixed, my bad sorry about that :face_in_clouds: and again thanks for your time